### PR TITLE
Wazo 2266 Removed queue_info table and references.

### PR DESCRIPTION
**Pull request by vamenard**

### CODE REVIEW

File: alembic/versions/203129d6b213_remove_queue_info_table.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+
+# revision identifiers, used by Alembic.
+revision = '203129d6b213'
+down_revision = 'ce76c220eaab'
+
+
+def upgrade():
+    op.drop_table('queue_info')
+    pass
```
Comment (bloom1) [inline alembic/versions/203129d6b213_remove_queue_info_table.py:19]: `pass` is only used when there is no body to a function: you can remove it (in both functions)

File: alembic/versions/203129d6b213_remove_queue_info_table.py [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+
+def downgrade():
+    op.create_table(
+        'queue_info',
+        Column('id', Integer, primary_key=True),
+        Column('queue_name', String(128), nullable=False, server_default='0'),
+        Column('caller', String(80), nullable=False, server_default='0'),
+        Column('caller_uniqueid', String(32), nullable=False, server_default='0'),
```
Comment (bloom1) [inline alembic/versions/203129d6b213_remove_queue_info_table.py:28]: For `queue_name`, `caller` and `caller_uniqueid`: `server_default` should be an empty string, not `0`.

File: alembic/versions/203129d6b213_remove_queue_info_table.py [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+def upgrade():
+    op.drop_table('queue_info')
+    pass
+
+
+def downgrade():
+    op.create_table(
+        'queue_info',
+        Column('id', Integer, primary_key=True),
+        Column('queue_name', String(128), nullable=False, server_default='0'),
```
Comment (bloom1) [inline alembic/versions/203129d6b213_remove_queue_info_table.py:26]: The `call_time_t` column is missing here

File: alembic/versions/203129d6b213_remove_queue_info_table.py [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+        Column('caller', String(80), nullable=False, server_default='0'),
+        Column('caller_uniqueid', String(32), nullable=False, server_default='0'),
+        Column('call_picker', String(80)),
+        Column('hold_time', Integer),
+        Column('talk_time', Integer),
+    )
+    pass
```
Comment (bloom1) [inline alembic/versions/203129d6b213_remove_queue_info_table.py:33]: You should also recreate the Indexes: (from the `xivo-dao` schema):
```python
class QueueInfo(Base):

    __tablename__ = 'queue_info'
    __table_args__ = (
        PrimaryKeyConstraint('id'),
        Index('queue_info_call_time_t_index', 'call_time_t'),
        Index('queue_info_queue_name_index', 'queue_name'),
# ...
```

**Review by bloom1 — CHANGES_REQUESTED** [review]
Great :smiley: if you have any questions don't hesitate

**Comment by vamenard** [comment]
recheck

**Review by bloom1 — APPROVED** [review]

**Pull request merged**